python/hopsworks/job.py

```diff
@@ -180,6 +193,49 @@ def delete(self):
         """
         self._job_api._delete(self)
 
+    def schedule(self, cron_expression, start_time=None, end_time=None):
```

Comment:
To be consistent with how we show what datatypes in the docs for client facing APIs - Can you remove the `datetime, optional` etc from the docstring below and instead add the type in the function signature? I.e like this https://github.com/logicalclocks/hopsworks-api/blob/14f46bca54f4f8f5eedd5c32d5cd3b86be590b17/python/hopsworks/job.py#L120

The resulting docs will show the type for each function parameter: https://docs.hopsworks.ai/hopsworks-api/3.4/generated/api/jobs/#run

Reply:
Fixed